public/diagnostic.js

```diff
@@ -23,3 +23,24 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Cart = Ember.Object.extend({
+
+  var orders: Ember.A(),
```

Comment:
No `var`. Also, it's not really necessary to create an Ember Array here - a normal array works just fine.